CHANGELOG.mkd

```diff
@@ -4,12 +4,15 @@ CHANGELOG
 Unreleased
 ----
 
+3.5.2
+-----
+
+- (RK-319) Clean up tmp directories used for downloading modules
+
 3.5.1
 -----
 
 - Upgrade Rugged Gemfile dependency for local development to ~> 1.0
-- Clean up tmp directories used for downloading modules
```

Comment:
Looks like the PR with this work was put up prior to 3.5.1, but merged after, so the changelog note got erroneously added to the 3.5.1 section, along with the rugged change, instead of staying in Unreleased.